### PR TITLE
IBM Power documentation updates for HCP

### PR DESCRIPTION
**Pull request by alishaIBM**
Issue: https://issues.redhat.com/browse/ACM-9778

### CODE REVIEW

File: clusters/hosted_control_planes/scale_nodepool_hosted_ibmpower.adoc [hunk 1]
```diff
@@ -120,27 +120,27 @@ See the following output:
 [source,bash]
```
Comment (swopebe) [inline clusters/hosted_control_planes/scale_nodepool_hosted_ibmpower.adoc:120]: I really would rather we remove these long outputs this release, if we can. We discussed calling out specifically what the user needs to know or see from the output, which in this case is `true`, `false` and maybe other info, but if the user sees this in the terminal, why do we need to repeat it in the doc.

We are trying to simplify to doc for less maintenance.

**Review by swopebe — APPROVED** [review]
Everything looks good but I do have the same comment as last release about the output.

**Comment by swopebe** [comment]
I am waiting on a peer review from my team and then this is ready to merge.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**